src/dmd/parse.d

```diff
@@ -7664,6 +7664,15 @@ final class Parser(AST) : Lexer
                 if (token.value == TOK.leftParentheses)
                 {
                     nextToken();
+                    if (token.value == TOK.identifier && peekNext() == TOK.leftParentheses)
+                    {
+                        error(loc, "unexpected `(` after `%s`, inside `is` expression. Try nesting the contents of `is` with a `typeof` expression", token.toChars());
```

Comment:
Should it be "Try *enclosing* the contents of \`is\` *in* a \`typeof\` expression"?